src/linux/py_proc.h

```diff
@@ -68,6 +68,21 @@ union {
 
 
 // ----------------------------------------------------------------------------
+static Elf64_Addr
+_py_proc_get_base64(Elf64_Ehdr * ehdr, void * elf_map)
+{
+  Elf64_Addr res = (Elf64_Addr) 0 - 1;
+  for (uint16_t i = 0; i < ehdr->e_phnum; ++i) {
+    Elf64_Phdr * phdr = (Elf64_Phdr *) (elf_map + i * ehdr->e_phoff);
```

Comment:
For this address computation, I would have expected something similar to `ELF_SH_OFF`, using both `e_phentsize` and `e_phoff`.

Reply:
Yes, this code was terribly broken.